lib/__tests__/index-test.js

```diff
@@ -353,7 +353,7 @@ describe('labels', () => {
 
   it('warns if an image without alt is the only content', () => {
     expectWarning(assertions.render.NO_LABEL.msg, () => {
-      <button><img src="#" alt=""/></button>;
+      <button><img src="#"/></button>;
```

Comment:
Right, I see. The next test has the same issue, in that case (referring to an empty alt attribute as "without alt"). In that case though, there should be additional tests for the `<button><img src="#" alt=""/></button>` case, right? Both "without alt" and "with empty alt" should be tested.